README.md

```diff
@@ -83,6 +90,34 @@ This step creates the following defaults:
 2. component files in the components folder called `pubsub.yaml`, `statestore.yaml` and `zipkin.yaml`. 
 3. default config file `$HOME/.dapr/config.yaml` for Linux/MacOS or for Windows at `%USERPROFILE%\.dapr\config.yaml` to enable tracing on `dapr init` call. Can be overridden with the `--config` flag on `dapr run`.
 
+#### Without Docker 
+
+```bash
+$ dapr init --slim
+⌛  Making the jump to hyperspace...
+creating default components folder: ~/.dapr/components
+↙  Downloading binaries and setting up components...
+removing archive ~/.dapr/placement_darwin_amd64.tar.gz
+
+installing Dapr to /usr/local/bin
+
+removing extracted binary ~/.dapr/placement
+↙  Downloading binaries and setting up components...
+removing archive ~/.dapr/daprd_darwin_amd64.tar.gz
+←  Downloading binaries and setting up components...
+installing Dapr to /usr/local/bin
+
+removing extracted binary ~/.dapr/daprd
+✅  Downloaded binaries and completed components set up.
+ℹ️  daprd binary has been installed.
+
+ℹ️  placement binary has been installed.
+
+✅  Success! Dapr is up and running. To get started, go here: https://aka.ms/dapr-getting-started
+```
+
+>Note: Only the Dapr runtime binary and the placement service binary are installed. An empty components folder, which is later used during `dapr run` unless the `--components-path` option is provided, is created. For Linux/MacOS, the default components folder path is `$HOME/.dapr/components` and for Windows it is `%USERPROFILE%\.dapr\components`.
```

Comment:
What is the scenario for creating am empty component folder?

Reply:
we set `standalone.DefaultComponentsDirPath()` as default and without an empty folder, it will throw an exception.